packages/core/test/validate_test.js

```diff
@@ -684,19 +643,15 @@ describe("Validation", () => {
           return errors;
         }
 
-        const { comp } = createFormComponent({
+        const { node, onError } = createFormComponent({
           schema,
           validate,
-          liveValidate: true,
-        });
-        comp.UNSAFE_componentWillReceiveProps({ formData });
-
-        expect(comp.state.errorSchema).eql({
-          0: { __errors: [] },
-          1: { __errors: [] },
-          2: { __errors: [] },
-          __errors: ["Forbidden value: bbb"],
+          formData,
```

Comment:
I see that you removed liveValidate for several of these tests. Can you add one test in which you use both liveValidate and a custom validate function, to make sure that functionality is still covered by our tests?